lib/couchdb/index.js

```diff
@@ -84,23 +85,47 @@ const upsert = R.curry((couch, bucket, keys, doc) =>
 )(doc)))
 
 
+const _findWhereEqById = R.curry((couch, bucket, predicates) =>
+  Bluebird.resolve({})
+  .then(() => getById(couch, bucket, predicates.id))
```

Comment:
Yep! I needed a `.tap` right before that to debug something and forgot to remove the `resolve` when i was done. Will do that when I'm in the office.